app/components/createContainer/createcontainer.html

```diff
@@ -350,6 +350,22 @@
                 </div>
               </div>
               <!-- !domainname -->
+              <!-- ipv4-input -->
+              <div class="form-group">
+                <label for="container_ipv4" class="col-sm-2 col-lg-1 control-label text-left">IPv4 Address</label>
+                <div class="col-sm-9">
+                  <input type="text" class="form-control" ng-model="formValues.IPv4" id="container_ipv4" placeholder="(Optional) e.g. xxx.xxx.xxx.xxx">
```

Comment:
Could you change the placeholder value to `e.g. 172.20.0.7` instead of `(Optional) e.g. xxx.xxx.xxx.xxx` ?